bot/commands/builtins.py

```
@@ -0,0 +1,357 @@
+                OUTPUT_DIR.iterdir(), key=lambda x: x.stat().st_mtime, reverse=True
+            )[:3]:
+                if item.is_dir():
+                    mtime = datetime.fromtimestamp(item.stat().st_mtime).strftime(
+                        "%Y-%m-%d %H:%M"
+                    )
+                    report = item / "report.md"
+                    summary = "No report"
+                    if report.exists():
+                        lines = report.read_text(encoding="utf-8").split("\n")
+                        summary = next(
+                            (l.strip("# ") for l in lines if l.startswith("# ")),
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)